recipes/commlib-py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "commlib-py" %}
+{% set version = "0.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/robotics-4-all/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 53b0906458dd1d6e38116a672dbefaf592cb8c8b67020f6f082dcf75b67960f1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --use-pep517 -vvv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry-core
+    - python >=3.7.0
+  run:
+    - hiredis >=2.1.1
+    - paho-mqtt >=1.6.1
+    - pika >=1.3.1
+    - poetry-core
```

Comment:
Is this really needed? Normally this is a build time only dependency.

Reply:
Right, it looks like a absent-minded copy-paste